modules/nf-core/gatk4/cnnscorevariants/environment.yml

```
@@ -0,0 +1,8 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/environment-schema.json
+channels:
+  - conda-forge
+  - bioconda
+dependencies:
+  # renovate: datasource=conda depName=bioconda/gatk4
+  - bioconda::gatk4=4.6.1.0
```

```suggestion
  - bioconda::gatk4=4.5.0.0
```


Because `cnnscorevariants` is deprecated in 4.6.1.0